README.md

```diff
@@ -153,14 +321,12 @@ More info: https://developers.rdstation.com/pt-BR/reference/contacts#methodPatch
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
 
-## Maintainer
-[Nando Sousa](mailto:fernando.sousa@resultadosdigitais.com.br)
+## Maintainers
 
-## Reference
+- [Nando Sousa](mailto:fernando.sousa@resultadosdigitais.com.br)
```

Comment:
@nandosousafr asked to remove him from this list of maintainers, could you do that?